games-strategy/dunelegacy/dunelegacy-0.96.4.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A Dune II game engine"
+DESCRIPTION="Dune Legacy is an effort by a handful of developers to revitalize \
+the first-ever real-time strategy game. \
+It tries to be as similar as possible to the original gameplay but to integrate \
+user interface features most modern realtime-strategy games have like selecting \
+multiple units"
+HOMEPAGE="https://dunelegacy.sourceforge.net/website/"
+COPYRIGHT="2016 Dune Legacy"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/dunelegacy/files/dunelegacy/$portVersion/dunelegacy-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="4740df9e94712b34f5d6fa1ca3c24658c4808b40431aa156ed66c318be9f4de8"
+SOURCE_FILENAME="dunelegacy-$portVersion-src.tar.bz2"
+PATCHES="dunelegacy-$portVersion.patchset"
+ADDITIONAL_FILES="dunelegacy.rdef.in"
+
+ARCHITECTURES="all ?x86_gcc2 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dunelegacy$secondaryArchSuffix = $portVersion
+	app:dunelegacy = $portVersion
```

Comment:
I'd prefer (my interpretations) to have the binary installed in $commandBinDir (try to look up some examples on how to do this).